lib/src/Base/Func/SymbolicEvaluation.cxx

```diff
@@ -112,6 +112,7 @@ Point SymbolicEvaluation::operator() (const Point & inP) const
 Sample SymbolicEvaluation::operator() (const Sample & inS) const
 {
   Sample result(parser_(inS));
+  result.setDescription(getOutputDescription());
   ++ callsNumber_;
```

Comment:
I just noticed that everywhere else we use callsNumber_ as the number of point evaluations:
```
$ grep -nr 'callsNumber_ +=' lib/src/
lib/src/Uncertainty/Algorithm/MetaModel/Kriging/KrigingGradient.cxx:158:  callsNumber_ += p;
lib/src/Uncertainty/Algorithm/MetaModel/Kriging/KrigingEvaluation.cxx:241:  callsNumber_ += size;
lib/src/Uncertainty/Algorithm/OrthogonalBasis/SoizeGhanemFactorEvaluation.cxx:141:  callsNumber_ += size;
lib/src/Base/Algo/KarhunenLoeveProjection.cxx:109:  callsNumber_ += outSample.getSize();
lib/src/Base/Func/FieldToPointFunctionImplementation.cxx:147:  callsNumber_ += size;
lib/src/Base/Func/BoxCoxEvaluation.cxx:141:  callsNumber_ += size;
lib/src/Base/Func/IdentityEvaluation.cxx:91:  callsNumber_ += inS.getSize();
lib/src/Base/Func/InverseBoxCoxEvaluation.cxx:136:  callsNumber_ += size;
lib/src/Base/Func/ProductPolynomialEvaluation.cxx:178:  callsNumber_ += size;
lib/src/Base/Func/LinearEvaluation.cxx:136:  callsNumber_ += size;
lib/src/Base/Func/P1LagrangeEvaluation.cxx:229:  callsNumber_ += size;
lib/src/Base/Func/ProductEvaluation.cxx:103:  callsNumber_ += size;
...
```